### ISSUE TITLE
Gas pipes wont work correctly

### ISSUE THREAD

**Issue by DroometYT** [pupnewfster/GasConduits#10]
#### Issue Description:
the gases won't transport through the gas pipe, from one tank/machine into another tank/machine. I know how to configure the outputs/inputs I play the mod quite a lot. It works with the mekanism pipes but not with the gas conduits.

#### Steps to reproduce:

1.connect gas conduit to gas tank and try to extract/insert into a empty gas tank

#### Versions (do not say latest):

- Gas Conduits:1.12.2-1.2.4
- EnderIO:1.12.2-5.2.61
- Mekanism:1.12.2-9.8.3.390
- Minecraft: 1.12.2
- Forge:14.23.5.2854

#### Your most recent log file where the issue was present: 

[pastebin/gist/etc link here]
(left top creative gas tank with hydrogen)
![image](https://user-images.githubusercontent.com/73135182/96498391-4aff6c80-124c-11eb-8381-bbec94436969.png)
![image](https://user-images.githubusercontent.com/73135182/96498480-62d6f080-124c-11eb-9260-03b47f6bdcec.png)

**Comment by pupnewfster** [comment]
Is the side config of the gas tanks setup to allow extraction from the side you are trying to extract and allow insertion into the side you are trying to insert into? And is the gas conduit set to pull or is there some redstone mode needed? (It has been a long time since I last touched 1.12 so I forget how ender io conduits work in general configuration wise)

**Comment by DroometYT** [comment]
yes it is, on the creative tank I did set every side to output and on the receiver tank every side to input 
I also tried to set the pipes in only one direction.

**Comment by DroometYT** [comment]
the normal conduits work fin without redstone or anything. It's just the gas pipes not transferring gas whatever configuration I use. I tried to fix it myself like for 2 hours and then started searching for another few hours
